arviz/data/__init__.py

```diff
@@ -1,8 +1,13 @@
-#pylint: disable=wildcard-import
+# pylint: disable=wildcard-import
```

Comment:
Theres no wildcard import here anymore so can't we get rid of this?

Reply:
I guess you are right!

Reply:
Ah this is merged, No worries I can made the change and upload a PR. Will do that right now

Reply:
Haha its no problem. This is a very minor thing.